### PR TITLE
add unlock email

### LINKED ISSUE #16: Add a new "unlock account" email
In order to implement the [account lockout functionality](https://github.com/mozilla/fxa-auth-server/issues/801#issuecomment-56592929) in the auth server, we will need to send out a new email.

https://wiki.mozilla.org/Identity/Firefox_Accounts/Account_lockout

### PR DESCRIPTION
**Pull request by shane-tomlinson**
A combination of @rfk's PR #17 and @rfeeley's copy.

fixes #16 

Goes along with:
* https://github.com/mozilla/fxa-auth-db-server/pull/123
* https://github.com/mozilla/fxa-auth-db-mem/pull/11
* https://github.com/mozilla/fxa-auth-db-mysql/pull/7
* https://github.com/mozilla/fxa-js-client/pull/141
* https://github.com/mozilla/fxa-content-server/pull/1806
* https://github.com/mozilla/fxa-auth-server/pull/867

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ module.exports = require('rc')(
       secure: false,
       sender: 'verification@accounts.firefox.com',
       verificationUrl: 'https://accounts.firefox.com/v1/verify_email',
-      passwordResetUrl: 'https://accounts.firefox.com/v1/complete_reset_password'
+      passwordResetUrl: 'https://accounts.firefox.com/v1/complete_reset_password',
+      accountUnlockUrl: 'https://accounts.firefox.com/v1/complete_unlock_account'
```
Comment (rfk) [inline config.js]: Does this page exist on the content-server side?

Comment (vladikoff) [inline config.js]: https://github.com/mozilla/fxa-content-server/pull/1806/files#diff-30dc249d0b58567a92b9ae045ac6ebc6R50

**Comment by shane-tomlinson** [comment]
@rfeeley - can you review the final copy of the email? https://github.com/shane-tomlinson/fxa-auth-mailer/blob/issue-16-unlock-email/templates/unlock.txt

@rfk or @chilts - mind doing a review?

File: templates/unlock.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                  <!-- BEGIN BODY // -->
+                  <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateBody" style="border-bottom:1px solid #CCCCCC;border-top:1px solid #FFFFFF;background-color:#FFFFFF;border-collapse:collapse !important;mso-table-rspace:0pt;mso-table-lspace:0pt;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%;">
+                    <tr>
+                      <td valign="top" class="bodyContent" mc:edit="body_content" style="text-align:center;padding-left:20px;padding-bottom:30px;padding-right:20px;padding-top:30px;line-height:150%;font-size:15px;font-family:Helvetica;color:#424F59;mso-table-rspace:0pt;mso-table-lspace:0pt;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%;">
+                        <h1 style="text-align:center;margin-left:0;margin-bottom:10px;margin-right:0;margin-top:0;letter-spacing:normal;line-height:100%;font-weight:normal;font-style:normal;font-size:26px;font-family:Helvetica;display:block;color:#424F59 !important;">{{t "Danger, Will Robinson!"}}</h1>
```
Comment (rfk) [inline templates/unlock.html]: "Danger, Will Robinson!" <-- probably we don't want to send this wording out to folks in production...

Comment (shane-tomlinson) [inline templates/unlock.html]: HA! I knew I should have used grep. Thanks for catching it.

**Comment by shane-tomlinson** [comment]
@rfk - "Danger, Will Robinson" text removed!

**Comment by rfk** [comment]
Hmm, I thought I left a note here yesterday but maybe it didn't save...anyway, this LGTM, r+ once we get rfeeley signoff on the text.

**Pull request merged**